### PR TITLE
Buyer can see the bids by clicking on the tender title - Aisha Athman Lali

### PR DESCRIPTION
**Pull request by aishaathmanlali**
## Naming Rules

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description

Write a summary of your change. Explain why you have made it.

## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes #1

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

File: client/src/SubmitBidForm.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { get, post } from "./TenderClient";
 import "./SubmitBidForm.css";
 import Logo from "./assets/images/CTY-logo-rectangle.png";
 
-const SubmitBidForm = () => {
+const SubmitBidForm = ({ onBidSubmitted }) => {
```
Comment (soonick) [inline client/src/SubmitBidForm.js]: What are you using `onSubmitted` for?

**Review by soonick — COMMENTED** [review]
I think there is a lot of code on this PR that shouldn't be there. Please clean it up

File: client/src/SubmitBidForm.js (hunk 1 above)
Comment (aishaathmanlali) [reply client/src/SubmitBidForm.js]: removed it

File: server/api.js [hunk 2]
```diff
@@ -748,6 +752,13 @@ router.post("/bid", async (req, res) => {
 
 			const bidResult = await client.query(bidQuery, bidValues);
 
+			const updateTenderQuery = `
+				UPDATE tender
+				SET no_of_bids_received = no_of_bids_received + 1
+				WHERE id = $1
+			`;
```
Comment (soonick) [inline server/api.js]: This doesn't belong here. Move it to another PR

File: server/api.js [hunk 3]
```diff
@@ -733,12 +736,13 @@ router.post("/bid", async (req, res) => {
 				return res.status(400).json({ code: "DUPLICATE_ENTRY" });
 			}
 			const bidQuery = `
-                      INSERT INTO bid (tender_id, bidder_id, bidding_date, status, bidding_amount, cover_letter, suggested_duration_days)
-					  VALUES ($1, $2, $3, $4, $5, $6, $7) RETURNING bid_id
+                      INSERT INTO bid (tender_id, bidder_id, buyer_id, bidding_date, status, bidding_amount, cover_letter, suggested_duration_days)
+					  VALUES ($1, $2, $3, $4, $5, $6, $7, $8) RETURNING bid_id
```
Comment (soonick) [inline server/api.js]: This shouldn't be necessary. I just noticed about this, but the bid table shouldn't have a `buyer_id` field. We know the buyer because we know the `tender_id`.

**Review by soonick — CHANGES_REQUESTED** [review]

File: server/api.js (hunk 2 above)
Comment (aishaathmanlali) [reply server/api.js]: This has already been merged. so I made a mistake because I opened a branch on top of the previous branch with this commit that has already been merged before this

File: server/api.js [hunk 4]
```diff
@@ -735,7 +734,7 @@ router.post("/bid", async (req, res) => {
 			}
 			const bidQuery = `
                       INSERT INTO bid (tender_id, bidder_id, bidding_date, status, bidding_amount, cover_letter, suggested_duration_days)
-					  VALUES ($1, $2, $3, $4, $5, $6, $7) RETURNING bid_id
+					  VALUES ($1, $2, $3, $4, $5, $6, $7, $8) RETURNING bid_id
```
Comment (soonick) [inline server/api.js]: Please. Test your code.

**Review by soonick — CHANGES_REQUESTED** [review]

File: server/api.js (hunk 4 above)
Comment (aishaathmanlali) [reply server/api.js]: it was working but I have corrected the mistake you pointed to

Comment (soonick) [reply server/api.js]: You need to test your code again if you change it

**Review by soonick — APPROVED** [review]
